### PR TITLE
Implement referent controllers and templates

### PR DESCRIPTION
**Pull request by tgalopin**

### CODE REVIEW

File: app/Resources/views/referent/layout.html.twig [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                                </li>
+                                <li {% block referent_menu_users_postal_code %}{% endblock %}>
+                                    <a href="{{ path('app_referent_users_postal_code') }}">
+                                        Envoyer un message par code postal
+                                    </a>
+                                </li>
+                            </ul>
+                        </li>
+                        <li>
+                            <h4>Evenements</h4>
```
Comment (hhamon) [inline app/Resources/views/referent/layout.html.twig]: Événements

File: src/AppBundle/Controller/ReferentController.php [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @Route("/referent")
+ * @Security("is_granted('ROLE_REFERENT')")
+ */
+class ReferentController extends Controller
+{
+    /**
+     * @Route("/utilisateurs", name="app_referent_users")
```
Comment (hhamon) [inline src/AppBundle/Controller/ReferentController.php]: Missing HTTP methods constraint.

File: src/AppBundle/Controller/ReferentController.php [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+     */
+    public function usersAction(): Response
+    {
+        // TODO IMPLEMENT
+        return $this->render('referent/users/list.html.twig');
+    }
+
+    /**
+     * @Route("/utilisateurs/envoyer-un-message/marcheurs", name="app_referent_users_subscribers")
```
Comment (hhamon) [inline src/AppBundle/Controller/ReferentController.php]: Missing HTTP methods constraint.

File: src/AppBundle/Controller/ReferentController.php [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+     */
+    public function usersSendMessageSubscribersAction(): Response
+    {
+        // TODO IMPLEMENT
+        return $this->render('referent/users/message-subscribers.html.twig');
+    }
+
+    /**
+     * @Route("/utilisateurs/envoyer-un-message/adherents", name="app_referent_users_adherents")
```
Comment (hhamon) [inline src/AppBundle/Controller/ReferentController.php]: Missing HTTP methods constraint.

File: src/AppBundle/Controller/ReferentController.php [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+     */
+    public function usersSendMessageAdherentsAction(): Response
+    {
+        // TODO IMPLEMENT
+        return $this->render('referent/users/message-adherents.html.twig');
+    }
+
+    /**
+     * @Route("/utilisateurs/envoyer-un-message/membres-comites", name="app_referent_users_followers")
```
Comment (hhamon) [inline src/AppBundle/Controller/ReferentController.php]: Missing HTTP methods constraint.

File: src/AppBundle/Controller/ReferentController.php [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+     */
+    public function usersSendMessageFollowersAction(): Response
+    {
+        // TODO IMPLEMENT
+        return $this->render('referent/users/message-followers.html.twig');
+    }
+
+    /**
+     * @Route("/utilisateurs/envoyer-un-message/animateurs-comites", name="app_referent_users_hosts")
```
Comment (hhamon) [inline src/AppBundle/Controller/ReferentController.php]: Missing HTTP methods constraint.

File: src/AppBundle/Controller/ReferentController.php [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+     */
+    public function usersSendMessageHostsAction(): Response
+    {
+        // TODO IMPLEMENT
+        return $this->render('referent/users/message-hosts.html.twig');
+    }
+
+    /**
+     * @Route("/utilisateurs/envoyer-un-message/code-postal", name="app_referent_users_postal_code")
```
Comment (hhamon) [inline src/AppBundle/Controller/ReferentController.php]: Missing HTTP methods constraint.

File: src/AppBundle/Controller/ReferentController.php [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+     */
+    public function usersSendMessagePostalCodeAction(): Response
+    {
+        // TODO IMPLEMENT
+        return $this->render('referent/users/message-postal-code.html.twig');
+    }
+
+    /**
+     * @Route("/evenements", name="app_referent_events")
```
Comment (hhamon) [inline src/AppBundle/Controller/ReferentController.php]: Missing HTTP methods constraint.

File: src/AppBundle/Controller/ReferentController.php [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+     */
+    public function eventsAction(): Response
+    {
+        // TODO IMPLEMENT
+        return $this->render('referent/events/list.html.twig');
+    }
+
+    /**
+     * @Route("/evenements/creer", name="app_referent_events_create")
```
Comment (hhamon) [inline src/AppBundle/Controller/ReferentController.php]: Missing HTTP methods constraint.

File: src/AppBundle/Controller/ReferentController.php [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+     */
+    public function eventsCreateAction(): Response
+    {
+        // TODO IMPLEMENT
+        return $this->render('referent/events/create.html.twig');
+    }
+
+    /**
+     * @Route("/comites", name="app_referent_commitees")
```
Comment (hhamon) [inline src/AppBundle/Controller/ReferentController.php]: Missing HTTP methods constraint.

**Comment by tgalopin** [comment]
Updated

**Pull request merged**